<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimePeriodDialogBase</class>
 <widget class="QDialog" name="TimePeriodDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select all data files matching a  time period and filters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="TextLabel_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>To:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="ToDate">
       <property name="displayFormat">
        <string>MM/dd/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ToDateCalendar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>29</horstretch>
         <verstretch>25</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select Ending date using calendar</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/gex/icons/calendar_icon.png</normaloff>:/gex/icons/calendar_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabelToDate">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;to date&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Files location</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel2">
        <property name="text">
         <string>Path:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LineEditPath">
        <property name="toolTip">
         <string>Path to STDF files to import</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonBrowsePath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>29</horstretch>
          <verstretch>25</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>29</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>29</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Browse disk to select a directory</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="CheckBoxSubfolders">
        <property name="text">
         <string>Look in subfolders</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="TextStatus">
     <property name="text">
      <string>&lt;files read/selected&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel1_4">
        <property name="text">
         <string>Tester name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="TesterNameFilter">
        <property name="toolTip">
         <string>Define tester name filter. e.g.: tester??_s</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel1">
        <property name="text">
         <string>Device name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="DeviceNameFilter">
        <property name="toolTip">
         <string>Define device name filter. e.g.: EX??RV03*</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel1_3">
        <property name="text">
         <string>Program name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ProgramNameFilter">
        <property name="toolTip">
         <string>Define program name filter. e.g.: GSM07T</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="TextLabel1_2">
        <property name="text">
         <string>Lot name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="LotNameFilter">
        <property name="toolTip">
         <string>Define lot name filter. e.g.: ERN023456T</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="TextLabel_2_2">
     <property name="text">
      <string>Filter strings can include '*' and '?' characters!</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonOk">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="TextLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>From:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="FromDate">
       <property name="displayFormat">
        <string>MM/dd/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FromDateCalendar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>29</horstretch>
         <verstretch>25</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select Starting date using calendar</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/gex/icons/calendar_icon.png</normaloff>:/gex/icons/calendar_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabelFromDate">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;from date&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QLabel" name="TextLabel_2_2_2">
     <property name="text">
      <string>Smart import: Query Data over a Time Period</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonOk_2">
     <property name="toolTip">
      <string>Abort import function</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>FromDateCalendar</tabstop>
  <tabstop>ToDateCalendar</tabstop>
  <tabstop>LineEditPath</tabstop>
  <tabstop>buttonBrowsePath</tabstop>
  <tabstop>CheckBoxSubfolders</tabstop>
  <tabstop>TesterNameFilter</tabstop>
  <tabstop>DeviceNameFilter</tabstop>
  <tabstop>ProgramNameFilter</tabstop>
  <tabstop>LotNameFilter</tabstop>
 </tabstops>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
